src/ActionPanel.scss

```diff
@@ -1,57 +1,91 @@
 @import 'variables';
 
-// TODO: Rework the arrow implementation. For example, it is not quite
-//       centered with respect to the action panel.
-
-.ActionPanel__panel {
-    width: $c2lc-action-panel-width;
+.ActionPanel__background {
+    align-self: center;
     height: $c2lc-action-panel-height;
-    background-color: #F1F1F1;
-    border: 0.1rem solid #939595;
-    border-radius: 3rem;
-    display: flex;
-    justify-content: space-evenly;
-    align-items: center;
-}
-
-.ActionPanel__panel::before {
-    content: '';
     position: absolute;
-    left: 45%;
-    top: 100%;
-    border-left: 0.7rem solid transparent;
-    border-top: 1rem solid #F1F1F1;
-    border-right: 0.7rem solid transparent;
+    top: -0.3rem;
+    width: $c2lc-action-panel-width;
     z-index: 1;
+
+    svg {
+        width: $c2lc-action-panel-width;
+        path {
+            fill: #30444E;
+            stroke: #4C9990;
+        }
+    }
 }
 
-.ActionPanel__panel::after {
-    content: '';
+.ActionPanel__panel {
+    align-items: center;
+    background-color: transparent;
+    display: flex;
+    height: $c2lc-action-panel-height;
+    justify-content: space-evenly;
+    padding-left: 2rem;
+    padding-right: 2rem;
     position: absolute;
-    left: 43.8%;
-    top: 100%;
-    border-left: 0.9rem solid transparent;
-    border-top: 1.2rem solid #939595;
-    border-right: 0.9rem solid transparent;
+    width: $c2lc-action-panel-width;
+    z-index: 2;
```

Comment:
Let's remove this z-index with the previous one.

Reply:
Done.